test/ros2_test.repos

```diff
@@ -0,0 +1,209 @@
+repositories:
```

Comment:
This is brittle, as this list is changing quickly. We should not need ament to compile a test package. Let's chat tomorrow, we should probably only need one package here, and we should be able to close a local URL (file://...)

Reply:
Updating to have E2E test use only demo_nodes package.